content/Hello yaaf - Juan Manuel Ramallo.md

```diff
@@ -0,0 +1,73 @@
+## Yet another active form
+
+Introducing [yaaf](https://github.com/rootstrap/yaaf), a gem to ease the usage of the form object pattern in rails apps.
+
+### ~The~ My pain points
+
+The form object pattern is widely used across Rails apps,
+and yet we tend to write these form objects in a different manner from project to project.
```

Comment:
What do you think about using the word way instead of manner here. Something in the lines of: _we tend to write these form objects in different ways from project to project_.

Reply:
Oh I like the word "manner" more than "way", I find it a bit more sophisticated, fancier than "way"